apps/sailfish/qml/pages/BrowserPage.qml

```diff
@@ -157,6 +157,9 @@ Page {
                         if(browserPage.model.hasDetails()) {
                             browserPage.model.fetchItemDetails(filterModel.mapToSourceIndex(listView.currentIndex));
                             drawer.open = true;
+                        } else {
+                            // no info, but we still want to be able to play/queue
+                            drawer.open = true;
```

Comment:
I think this needs to check if the item is actually playable? So:
```Javascript
} else if (playable) {
  drawer.open  = true;
}
```

Otherwise it should be changed to just:
```JavaScript
drawer.open = true;
if(browserPage.model.hasDetails()) {
    browserPage.model.fetchItemDetails(filterModel.mapToSourceIndex(listView.currentIndex));
}
```

And it might be better to use a simple `ContextMenu` instead of showing an expanded item which is empty (only has the `PullDownMenu`).
@mzanetti how do you handle this in Ubuntu (reading the code I think you also don't handle this case)?